src/Resources/views/storefront/component/payment/payment-action-modal.html.twig

```diff
@@ -8,7 +8,7 @@
         <div class="modal-content">
             <div class="modal-body">
                 <div data-adyen-payment-action-container>
-                    <div class="adyen-payment-action-container loader" role="status">
+                    <div class="adyen-payment-container loader" role="status">
```

Comment:
what happens if the 3DS2 modal is closed and we want to retry payment?
I assume in this case if the modal is the same as the modal including the input fields before the action then we won't be able to do so and the page needs to be refreshed manually.

Could you check this scenario if it handles it well?

Reply:
@cyattilakiss there is no 'close' button for the 3ds2 modal and it also does not close when you click outside the modal, the only way to close the modal is to refresh the page, since the order has already been created and the cart has been cleared, this will lead to losing the payment session altogether. Payment will remain as 'in progress' and cannot be completed, until OFFER_CLOSED notification sets the payment as 'failed'. 
Also, there is a different modal `payment-component-modal` with the input fields for the payment data